tests/integ/test_workflow.py

```diff
@@ -855,6 +855,113 @@ def test_one_step_callback_pipeline(sagemaker_session, role, pipeline_name, regi
             pass
 
 
+def test_steps_with_map_params_pipeline(
```

Comment:
I might be missing it but how is this testing map params?

Reply:
assert at line 947 and 948 are checking hyperparameter values? Do we need different asserts?

Reply:
Updated for testing hyperparameters